exampleData.js

```diff
@@ -0,0 +1,25 @@
+const faker = require("faker");
+
+console.log(faker.lorem.sentences((min = 1), (max = 5)));
+var user = [];
+
+for (let i = 0; i < 100; i++) {
+  let userInfo = {
+    restaurantId: i,
+    name: faker.name.findName(),
+    location: faker.address.city(),
+    numFriends: faker.random.number({ min: 1, max: 500 }),
+    numReviews: faker.random.number({ min: 1, max: 500 }),
+    numPhotos: faker.random.number({ min: 1, max: 501 }),
+    elite: faker.random.boolean(),
+    stars: faker.random.number({ min: 1, max: 5 }),
+    date: faker.date.between("2015-01-01", "2019-01-05"),
```

Comment:
Nice! I need to refactor my date generation. I didn't know about the faker.date.between. I'm going to copy that!

Reply:
@kimbyongjin its nice but i still need to figure out how to show only the date. it seems like faker likes to add time to it